test/parallel/test-http2-connect-method.js

```diff
@@ -53,6 +55,40 @@ server.listen(0, common.mustCall(() => {
   proxy.listen(0, () => {
     const client = http2.connect(`http://localhost:${proxy.address().port}`);
 
+    // confirm that :authority is required and :scheme & :path are forbidden
+    assert.throws(
+      () => client.request({
+        [HTTP2_HEADER_METHOD]: 'CONNECT'
+      }),
+      common.expectsError({
```

Comment:
Just a note for a future test: you can write `common.expectsError(throwingFn, errorObj)` instead of `assert.throws(throwingFn, common.expectsError(errorObj))`.

Reply:
Ah, thanks @BridgeAR! Will use in the future.